README.md

```diff
@@ -133,6 +133,37 @@ A session can be ended using following method call:
 `convivaAnalyticsIntegration.endSession()`
 Since the `BitmovinPlayer` automatically pauses the video if no background playback is configured the session creation after the app is in foreground again is handled automatically.
 
+#### Server Side Ad Tracking
```

Comment:
nit: Why is `####` is used under `##` right away?

Reply:
Good question, I just aligned with surrounding headlines.

Reply:
[386d858](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/76/commits/386d858b80bedb774e519f4057a9ca4843eb177c)